pom.xml

```diff
@@ -1,179 +1,206 @@
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-jodatime/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-security-web/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-codepoints/src/main/java</sourceDirectory>
-                                    <sourceDirectory>terasoluna-gfw-common-libraries/terasoluna-gfw-string/src/main/java</sourceDirectory>
-                                </sourceDirectories>
-                            </configuration>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
-            </build>
-        </profile>
-    </profiles>
+  </organization>
+  <developers>
+    <developer>
+      <name>NTT DATA</name>
+      <organization>NTT DATA Corporation</organization>
+      <organizationUrl>http://terasolunaorg.github.io/</organizationUrl>
+    </developer>
+  </developers>
+  <scm>
+    <connection>scm:git:git@github.com:terasolunaorg/terasoluna-gfw.git</connection>
+    <developerConnection>scm:git:git@github.com:terasolunaorg/terasoluna-gfw.git</developerConnection>
+    <url>git@github.com:terasolunaorg/terasoluna-gfw.git</url>
+  </scm>
+  <properties>
+    <!-- == Maven Plugin Versions == -->
+    <jacoco-maven-plugin.version>0.7.5.201505241946</jacoco-maven-plugin.version>
+    <formatter-maven-plugin.version>2.0.1</formatter-maven-plugin.version>
+    <xml-maven-plugin.version>1.0.1</xml-maven-plugin.version>
+    <com.google.code.maven-license-plugin.version>1.4.0</com.google.code.maven-license-plugin.version>
+    <coveralls-maven-plugin.version>4.1.0</coveralls-maven-plugin.version>
+    <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
+    <nexus-staging-maven-plugin.version>1.6.7</nexus-staging-maven-plugin.version>
+    <!-- == Project Properties == -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.deploy.skip>true</maven.deploy.skip>
+  </properties>
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>xml-maven-plugin</artifactId>
+          <version>${xml-maven-plugin.version}</version>
+          <configuration>
+            <indentSize>2</indentSize>
```

Comment:
In maven, it is possible to process only a specific module by attaching the -pl option.
Fix to prevent the format check from being executed with the default setting of terasoluna-gfw-parent etc. by specifying the terasoluna-gfw/pom.xml with the -pl option.
Also, at the terasoluna-gfw/pom.xl , I added <include> and <exclude> property to check all appropriate xml files in terasoluna-gfw .

I changed at c79fcdaebc1ce6674e08090844c701f4b71bcf73 commit .